templates/documentation/delivery-analytics/player-customization.md

```diff
@@ -1,12 +1,11 @@
 ---
-title: "Player customization"
-slug: "player-customization"
+title: api.video Player customization
+slug: player-customization
```

Comment:
`slug` can be removed, as confirmed by Doctave [here](https://api-video.slack.com/archives/C05DTNR3QDS/p1697618392282579)

Reply:
Done ✅